package.json

```diff
@@ -21,6 +21,9 @@
     "license": "ts-node scripts/license-headers",
     "start": "start-storybook -p 6006",
     "test": "jest",
+    "test:dev": "TEST_ENVIRONMENT=dev jest",
+    "test:staging": "TEST_ENVIRONMENT=staging jest",
+    "test:production": "TEST_ENVIRONMENT=production jest",
```

Comment:
Here you probably want to use cross-env so that this also works correctly on Windows.